<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PublisherGui</class>
 <widget class="QMainWindow" name="PublisherGui">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>650</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>100</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>2000</width>
    <height>2000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>./icons/PublishIcon1.png</normaloff>./icons/PublishIcon1.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(33, 34, 38);
</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="PublisherWindow">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>2000</width>
     <height>2000</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="font">
       <font>
        <family>Punishment</family>
        <pointsize>44</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>THE PUBLISHER</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="projectCB">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>2000</width>
        <height>2000</height>
       </size>
      </property>
      <property name="toolTip">
       <string>SELECT PROJECT</string>
      </property>
      <property name="statusTip">
       <string/>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(150, 150, 150);</string>
      </property>
      <item>
       <property name="text">
        <string>SELECT PROJECT</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="renderSelectCB">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>2000</width>
        <height>2000</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(150, 150, 150);</string>
      </property>
      <item>
       <property name="text">
        <string>SELECT RENDER</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="exploreImgBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>2000</width>
        <height>2000</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift Condensed</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="toolTip">
       <string>Open render folder in file browser</string>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>EXPLORE RENDER PATH</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="viewImgBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>2000</width>
        <height>2000</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift Condensed</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>VIEW RENDER</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="sequenceSelectCB">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>2000</width>
        <height>2000</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(150, 150, 150);</string>
      </property>
      <item>
       <property name="text">
        <string>SELECT A SEQUENCE</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="shotsCB">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>2000</width>
        <height>2000</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(150, 150, 150);</string>
      </property>
      <item>
       <property name="text">
        <string>SELECT SHOT</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="exploreOutImgBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>2000</width>
        <height>2000</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift Condensed</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>EXPLORE OUTPUT</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="moveRenBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>2000</width>
        <height>2000</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift Condensed</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>PUBLISH RENDER</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="moveRenPB">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>2000</width>
        <height>2000</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(60, 26, 255);</string>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="publishBtn">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>2000</width>
        <height>2000</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift Condensed</family>
        <pointsize>30</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(54, 57, 64);
image: url(:/buttonImages/skullBullets_white.png);
</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>./icons/skullBullets_white.png</normaloff>./icons/skullBullets_white.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="publishBtnPB">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>2000</width>
        <height>2000</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(255, 39, 245);</string>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>projectCB</zorder>
   <zorder>renderSelectCB</zorder>
   <zorder>sequenceSelectCB</zorder>
   <zorder>shotsCB</zorder>
   <zorder>viewImgBtn</zorder>
   <zorder>moveRenBtn</zorder>
   <zorder>exploreOutImgBtn</zorder>
   <zorder>moveRenPB</zorder>
   <zorder>publishBtnPB</zorder>
   <zorder>publishBtn</zorder>
   <zorder>exploreImgBtn</zorder>
   <zorder>titleLabel</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionStudio_Settings"/>
   </widget>
   <widget class="QMenu" name="menuPlugins">
    <property name="title">
     <string>Plugins</string>
    </property>
    <addaction name="actionPlayer_settings"/>
    <addaction name="actionKitsu"/>
   </widget>
   <addaction name="menuSettings"/>
   <addaction name="menuPlugins"/>
  </widget>
  <action name="actionFile">
   <property name="text">
    <string>File</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionStudio_Settings">
   <property name="text">
    <string>Studio Settings</string>
   </property>
  </action>
  <action name="actionPlayer_settings">
   <property name="text">
    <string>Player settings</string>
   </property>
  </action>
  <action name="actionKitsu">
   <property name="text">
    <string>Kitsu</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../icons/resources.qrc"/>
  <include location="../fonts/fonts.qrc"/>
 </resources>
 <connections/>
</ui>
